### PR TITLE
Fix #1676: AdministratorControl orientation change bug

### LINKED ISSUE #1676: AdministratorControl orientation change bug
**Describe the bug**
App crash on Administrator screen when the screen rotates.

**Possible cause**
Null fragment reference in AdministratorControlsAccountActionsViewModel after orientation changes

**To Reproduce**
Steps to reproduce the behaviour:
1. Open Administrator control screen
2. Click on logout -> see dialog appears -> click on cancel
3. rotate the screen -> click on logout -> app crash

**Expected behaviour**
Dialog should appear

**Error**
```
2020-08-19 12:58:02.702 1522-1522/org.oppia.app E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.oppia.app, PID: 1522
    kotlin.KotlinNullPointerException
        at org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel.AdministratorControlsAccountActionsViewModel.onLogOutClicked(AdministratorControlsAccountActionsViewModel.kt:15)
        at org.oppia.app.databinding.AdministratorControlsAccountActionsViewBindingLandImpl._internalCallbackOnClick(AdministratorControlsAccountActionsViewBindingLandImpl.java:134)
        at org.oppia.app.generated.callback.OnClickListener.onClick(OnClickListener.java:11)
        at android.view.View.performClick(View.java:6597)
        at android.view.View.performClickInternal(View.java:6574)
        at android.view.View.access$3100(View.java:778)
        at android.view.View$PerformClick.run(View.java:25885)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix #1676 

### Test passes with fix 
Screenshot 2020-08-23 at 17 16 17

### Test fail without fix 
Screenshot 2020-08-23 at 17 10 09

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [ ] The PR is made from a branch that is up-to-date with "develop".
- [ ] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — COMMENTED** [review]
This does work correctly. But will need to defer to @BenHenning about approach.

**Comment by anandwana001** [comment]
> This does work correctly. But will need to defer to @BenHenning about approach.

Thanks for having a look at this, might my save time to complete this asap.

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 1]
```diff
@@ -441,6 +441,34 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_changeConfiguration_clickLogout_displaysLogoutDialog() {
+    ActivityScenario.launch<AdministratorControlsActivity>(
+      createAdministratorControlsActivityIntent(
+        0
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Please add a named argument to qualify this 0 value, e.g. ``profileId = 0``. This helps provide context to readers.

**Review by BenHenning — APPROVED** [review]
Thanks @anandwana001. Fix LGTM! Added one comment--feel free to resolve it after addressing the comment. No need to reassign me.

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Done

**Pull request merged**